### PR TITLE
added coverage for overTimeLoader

### PR DESCRIPTION
**Pull request by Carla-Moz**
Also fixed a bug re getting interval info from url while writing tests for overTimeLoader.

### CODE REVIEW

File: src/components/CompareResults/overTimeLoader.ts [hunk 1]
```diff
@@ -26,10 +26,30 @@ function checkValues({
...
+    );
+  }
+
   if (framework === null) {
     framework = 1; // default to talos so that manually typing the URL is easier
   }
 
+  if (interval === null) {
+    interval = 86400; // default to 1 day
+  }
```
Comment (julienw) [inline src/components/CompareResults/overTimeLoader.ts:51]: for compare over time, shouldn't this be an error actually?

File: src/components/CompareResults/overTimeLoader.ts [hunk 2]
```diff
@@ -46,9 +66,6 @@ function checkValues({
     );
   }
 
-  if (interval === null) {
-    interval = 86400; // default to 1 day
-  }
   const intervalValue = +interval as TimeRange['value'];
```
Comment (julienw) [inline src/components/CompareResults/overTimeLoader.ts:69]: I just noticed that the return values below do not return these values, but instead they return hardcoded values :) This should be fixed (in this PR or in another PR as you wish)

**Review by julienw — APPROVED** [review]
thanks for the additional coverage

File: src/components/CompareResults/overTimeLoader.ts (hunk 2 above)
Comment (Carla-Moz) [reply src/components/CompareResults/overTimeLoader.ts:69]: That's silly. I don't know why I did that. I must have been testing something and forgot to revert it. I'll fix it in this PR. Thanks for catching that!

**Pull request merged**